### ISSUE TITLE
Double tap doesn't switch screen from Kubuntu server to Win10 client

### ISSUE THREAD

**Issue by database9-445** [symless/synergy-core#5011]
This bug report applies to the nightly version of Synergy. It does not apply to the stable version.

I have a Kubuntu (KDE) computer acting as a Synergy server. I have a Windows 10 computer acting as a client. Both computers were initially running the nightly version of Synergy that I downloaded on August 23. 

Double-tapping on the edge of the server screen (Kubuntu) should have caused the cursor to move to the client screen. It did not. However, double-tapping on the client screen did cause the cursor to switch to the server. Moving window around had no effect. Minimizing or maximizing windows had no effect. Moving the cursor to the bottom or top corner of the screen had no effect.

This problem continued after I downloaded and installed the stable version of Synergy (1.7.4) on the server. It cleared up when I installed the stable version on both the client and the server.  I suspect a regression has been introduced.

**Comment by database9-445** [comment]
It's now showing the same behaviour in the stable version. I have TeamViewer running on both computers and I think it's interfering with Synergy. Still trying to isolate this.

**Comment by database9-445** [comment]
It's been a while since I first encountered this issue. I've noticed another glitch that seems related. About half the time I cannot use the clipboard to copy from the Kubuntu machine to the Windows 10 machine.  In order to get the clipboard working again I have to completely shut down TeamViewer on both machines and also restart Synergy on both machines. Sometimes the clipboard issue and the scroll locking coincide. Sometimes not. I haven't been able to isolate exactly what triggers the problem.
